SU2_CFD/src/fluid/CDataDrivenFluid.cpp

```diff
@@ -268,37 +338,37 @@ void CDataDrivenFluid::Evaluate_Dataset(su2double rho, su2double e) {
   }
 }
 
-void CDataDrivenFluid::Run_Newton_Solver(su2double Y1_target, su2double Y2_target, su2double* Y1, su2double* Y2,
-                                         su2double* dY1drho, su2double* dY1de, su2double* dY2drho, su2double* dY2de) {
+void CDataDrivenFluid::Run_Newton_Solver(const su2double Y1_target, const su2double Y2_target, su2double const & Y1, su2double const & Y2,
+                                         su2double const & dY1drho, su2double const & dY1de, su2double const & dY2drho, su2double const & dY2de) {
```

Comment:
etc.
```suggestion
                                         const su2double& dY1drho, su2double const & dY1de, su2double const & dY2drho, su2double const & dY2de) {
```

Reply:
The terms dY1drho, dY2drho, etc should remain modifiable. That's why they are defined as constant references rather than references to constants.

Reply:
That's not the way it works